src/flash/flasher.cpp

```diff
@@ -11,6 +11,9 @@
 
 PING_LOGGING_CATEGORY(FLASH, "ping.flash")
 
+namespace {
+    const QVector<int> validBaudRates = {57600, 115200, 230400};
+}
```

Comment:
Please add a space between the namespace and the class constructor.
Why are you moving from privated to namespace ?
Please, be more clear in your commit.

Reply:
I'll rework the commit messages today. anything that's private / static should go into the cpp's unammed namespace, this will reduce the compilation time for the program.